homeassistant/components/cloudflare/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+    "flow_title": "Cloudflare: {name}",
+    "step": {
+      "user": {
+        "title": "Connect to Cloudflare",
+        "data": {
+          "api_token": "[%key:common::config_flow::data::api_token%]"
```

Comment:
What kind of API token should be used here?

![Uploading Screen Shot 2020-10-18 at 2.01.58 PM.png…]()

Reply:
Edit zone DNS

Reply:
I guess we could mention that in description of user flow like we do in docs?

Reply:
That would be great as it was a hard stop on the user experience without the information.